bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/extensions/PersistenceExtensions.java

```diff
@@ -124,52 +289,89 @@ public static void persist(Item item) {
             } else {
                 return null;
             }
-        } else {
-            LoggerFactory.getLogger(PersistenceExtensions.class)
-                    .warn("There is no queryable persistence service registered with the id '{}'", serviceId);
-            return null;
         }
```

Comment:
I would prefer to keep the `else` branch, it is more clear (also applies to other methods).

Reply:
OK